docs/howto/module_pattern.md

```diff
@@ -0,0 +1,285 @@
+---
+id: howto_module_pattern
+title:  "Module pattern"
```

Comment:
How about having a title starting with a verb so that it sounds good when you read "How to ...". I think all the other sections are like that.